pkg/build/api/types.go

```diff
@@ -242,6 +266,10 @@ const (
 	// GenericWebHookType represents a trigger that launches builds on
 	// generic webhook invocations
 	GenericWebHookType BuildTriggerType = "generic"
+
+	// ImageChangeType represents a trigger that launches builds on
+	// availability of a new version of an image
+	ImageChangeType BuildTriggerType = "imageChange"
```

Comment:
This constant is misnamed - all constants should include the type, so "ImageChangeBuildTriggerType" and "GenericWebHookBuildTriggerType".  Please fix the constant name (but not the constant itself, that's fine)